src/models/callercalleemodel.h

```diff
@@ -201,22 +214,32 @@ class LocationCostModelImpl : public HashModel<Data::LocationCostMap, ModelImpl>
         if (role == Qt::InitialSortOrderRole && column == Cost) {
             return Qt::DescendingOrder;
         }
-        if (role != Qt::DisplayRole) {
+
+        if (role != Qt::DisplayRole && role != Qt::ToolTipRole) {
             return {};
         }
 
-        switch (column) {
-            case Location:
-                return QObject::tr("Location");
-            case Cost:
-                return QObject::tr("Cost");
+        if (role == Qt::DisplayRole) {
+            switch (column) {
+                case Location:
+                    return QObject::tr("Location");
+                case Cost:
+                    return QObject::tr("Cost");
+            }
+        } else if (role == Qt::ToolTipRole) {
+            switch (column) {
+                case Location:
+                    return QObject::tr("The file name and line number of the source file where this this symbol resides. May be empty when debug information is missing.");
```

Comment:
The source file name and line number where the cost was measured. May be empty when debug information is missing.